pkgs/development/python-modules/pygtfs/default.nix

```diff
@@ -3,42 +3,34 @@
   buildPythonPackage,
   docopt,
   fetchPypi,
-  nose,
   pytz,
-  pythonOlder,
   setuptools-scm,
   six,
   sqlalchemy,
+  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "pygtfs";
   version = "0.1.9";
-  format = "setuptools";
-
-  disabled = pythonOlder "3.7";
+  pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
     hash = "sha256-J5vu51OOMabWd8h60PpvvBiCnwQlhEnBywNXxy9hOuA=";
   };
 
-  postPatch = ''
-    # https://github.com/jarondl/pygtfs/pull/72
-    substituteInPlace setup.py \
-      --replace "pytz>=2012d" "pytz"
-  '';
-
   nativeBuildInputs = [ setuptools-scm ];
```

Comment:
This should be in `build-system` (and you might want to list `setuptools` itself as well to be explicit, but it’s not required).